### PR TITLE
💄 Update QA comments

### PR DESCRIPTION
**Pull request by AuroraHuang22**
1. [update copywriting](https://www.notion.so/likecoin/d858f215bf1f47d29e576c9e09cdb5b5) 
2. [update transfer tooltip](https://www.notion.so/likecoin/Cancel-transfer-copywriting-4b46570c94fd403c82eacf1017d8c859) & [icon](https://www.notion.so/likecoin/Update-transfer-popup-icon-32a74dabfe4a48d188029facff4a5ebf)
3. [Disable send NFT to myself](https://www.notion.so/likecoin/can-sent-to-the-same-account-donno-if-it-will-cause-spamming-4bd87cf283c646a28c94831b9381174f)
4. [Fix reset likerInfo](https://www.notion.so/likecoin/sticky-LikerID-avatar-e0dedd2ecaac49efb36b74e3cc1e66c2)
5. [Update keplr-mobile Icon](https://www.notion.so/likecoin/Stories-grouped-by-assignee-d4c81c5e259c4cc48593f86885f15975?p=d858f215bf1f47d29e576c9e09cdb5b5&pm=s#:~:text=%40JoshKiu%20Lo%20%E6%83%B3%E8%AB%8B%E4%BD%A0%E5%B9%AB%E6%88%91%E5%87%BA%20keplr%2Dmobile%20%E7%9A%84%20icon%20%2C%20%E7%94%A8%E5%9C%A8%20sign%20attention%20%E7%9A%84)
6. Log events

### CODE REVIEW

**Review by nwingt — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Comment by nwingt** [comment]
Please rebase

File: src/mixins/nft.js [hunk 1]
```diff
@@ -262,12 +278,20 @@ export default {
...
+            this.getAddress,
+            1
+          );
           this.uiSetTxError('INSUFFICIENT_BALANCE');
           this.uiSetTxStatus(TX_STATUS.INSUFFICIENT);
           return;
         }
 
         this.uiSetTxStatus(TX_STATUS.SIGN);
+        logTrackerEvent(this, 'NFT', 'NFTTransfer-sendGrant', this.classId, 1);
```
Comment (nwingt) [inline src/mixins/nft.js:294]: Maybe put the event to the page instead of this mixin, so that we can know where the event triggered from

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/mixins/nft.js [hunk 2]
```diff
@@ -227,19 +228,34 @@ export default {
         this.uiToggleCollectModal();
         this.uiSetCollectedCount(this.userOwnedCount);
         if (balance === '0' || Number(balance) < this.purchaseInfo.totalPrice) {
+          logTrackerEvent(
+            this,
+            'NFT',
+            'NFTCollect-insufficient',
```
Comment (nwingt) [inline src/mixins/nft.js:234]:
```suggestion
            'NFTCollectErrorNoBalance',
```

File: src/mixins/nft.js [hunk 3]
```diff
@@ -227,19 +228,34 @@ export default {
...
           return;
         }
 
         this.uiSetTxStatus(TX_STATUS.SIGN);
+        logTrackerEvent(this, 'NFT', 'NFTCollect-sendGrant', this.classId, 1);
         const txHash = await sendGrant({
           senderAddress: this.getAddress,
           amountInLIKE: this.purchaseInfo.totalPrice,
           signer: this.getSigner,
         });
```
Comment (nwingt) [inline src/mixins/nft.js:249]: Add logs around the request
```suggestion
        logTrackerEvent(this, 'NFT', 'NFTCollectSendGrantRequested', this.classId, 1);
        const txHash = await sendGrant({
          senderAddress: this.getAddress,
          amountInLIKE: this.purchaseInfo.totalPrice,
          signer: this.getSigner,
        });
       logTrackerEvent(this, 'NFT', 'NFTCollectSendGrantApproved'', this.classId, 1);
```

File: src/mixins/nft.js [hunk 4]
```diff
@@ -227,19 +228,34 @@ export default {
...
 
         if (txHash && this.uiIsOpenCollectModal) {
+          logTrackerEvent(
+            this,
+            'NFT',
+            'NFTCollect-postNFTPurchase',
```
Comment (nwingt) [inline src/mixins/nft.js:255]:
```suggestion
            'NFTCollectPurchase',
```

File: src/mixins/nft.js [hunk 5]
```diff
@@ -248,7 +264,7 @@ export default {
           this.uiSetTxStatus(TX_STATUS.COMPLETED);
```
Comment (nwingt) [inline src/mixins/nft.js:264]: Add another log here
```suggestion
          this.uiSetTxStatus(TX_STATUS.COMPLETED);
          logTrackerEvent(this, 'NFT', 'NFTCollectPurchaseCompleted'', this.classId, 1);
```

File: src/mixins/nft.js [hunk 6]
```diff
@@ -262,12 +278,20 @@ export default {
         await this.initIfNecessary();
         const balance = await getAccountBalance(this.getAddress);
         if (balance === '0') {
+          logTrackerEvent(
+            this,
+            'NFT',
+            'NFTTransfer-insufficient',
```
Comment (nwingt) [inline src/mixins/nft.js:284]:
```suggestion
            'NFTTransferErrorNoBalance',
```

**Review by nwingt — COMMENTED** [review]
Some suggestion for logs

File: src/mixins/nft.js (hunk 1 above)
Comment (AuroraHuang22) [reply src/mixins/nft.js:294]: We have logged the event everywhere before `collectNFT()` is triggered, so I think we can know where this `sendGrand` event came from

[campaign page](https://github.com/likecoin/liker-land/blob/master/src/components/NFTCampaign/Item.vue#:~:text=handleClickCollect()%20%7B,%7D%2C)
[portfolio page](https://github.com/likecoin/liker-land/blob/master/src/components/NFTCampaign/Item.vue#:~:text=handleClickCollect()%20%7B,%7D%2C)
[details page](https://github.com/likecoin/liker-land/blob/master/src/pages/nft/class/_classId.vue#:~:text=%7D%2C-,handleCollectFromPreviewSection()%20%7B,%7D%2C,-async%20getLIKEPrice()%20%7B)

**Review by nwingt — APPROVED** [review]

**Pull request merged**